guava-testlib/pom.xml

```diff
@@ -3,9 +3,9 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>com.google.guava</groupId>
-    <artifactId>guava-parent</artifactId>
-    <version>24.1-jre</version>
+    <groupId>com.github.lafa.cache</groupId>
+    <artifactId>cache-parent</artifactId>
+    <version>1.0</version>
```

Comment:
can you make the version major.minor.patch ? 
also up the version, for every master merge we will try to push and the push will fail. the public maven repository does not allow same version push